crates/typos-lsp/tests/integration_test.rs

```diff
@@ -325,6 +325,19 @@ async fn test_ignore_typos_in_config_files() {
     );
 }
 
+#[test_log::test(tokio::test)]
+async fn test_newer_dict() {
+    let did_open = did_open("heaby");
+
+    let mut server = TestServer::new();
+    let _ = server.request(&initialize()).await;
+
+    similar_asserts::assert_eq!(
+        server.request(&did_open).await,
+        publish_diagnostics(&[diag("`heaby` should be `heavy`", 0, 0, 5)])
+    );
+}
+
```

Comment:
If you don't mind i'll remove this - it's good to verify we have the latest dict, but probably isn't useful as an ongoing regression test ie: we don't need to protect against the dictionary is rolled back to an earlier version.
```suggestion
```

Reply:
Agreed, this test is added only for testing this PR. I'm okay to removing :+1: